python/src/aiconfig/editor/client/src/components/prompt/PromptActionBar.tsx

```diff
@@ -44,48 +45,60 @@ export default memo(function PromptActionBar({
-          <Tabs defaultValue="settings">
-            <Tabs.List>
-              <Tabs.Tab value="settings">Settings</Tabs.Tab>
-              {checkParametersSupported(prompt) && (
-                <Tabs.Tab value="parameters">
-                  Local Variables (Parameters)
-                </Tabs.Tab>
-              )}
-            </Tabs.List>
+        <>
+          <Container miw="400px">
+            <ActionIcon
+              size="sm"
+              onClick={() => setIsExpanded(false)}
+              mt="0.5em"
+            >
+              <IconClearAll />
+            </ActionIcon>
+            <Tabs defaultValue="settings" mb="1em">
+              <Tabs.List>
+                <Tabs.Tab value="settings">Settings</Tabs.Tab>
+                {checkParametersSupported(prompt) && (
+                  <Tabs.Tab value="parameters">
+                    Local Variables (Parameters)
+                  </Tabs.Tab>
+                )}
+              </Tabs.List>
 
-            <Tabs.Panel value="settings">
-              <ModelSettingsRenderer
-                settings={getModelSettings(prompt)}
-                schema={modelSettingsSchema}
-                onUpdateModelSettings={onUpdateModelSettings}
-              />
-              <PromptMetadataRenderer
-                prompt={prompt}
-                schema={promptMetadataSchema}
-              />
-            </Tabs.Panel>
-
-            {checkParametersSupported(prompt) && (
-              <Tabs.Panel value="parameters">
-                <ParametersRenderer
-                  initialValue={getPromptParameters(prompt)}
-                  onUpdateParameters={onUpdateParameters}
+              <Tabs.Panel value="settings">
+                <ModelSettingsRenderer
+                  settings={getModelSettings(prompt)}
+                  schema={modelSettingsSchema}
+                  onUpdateModelSettings={onUpdateModelSettings}
+                />
+                <PromptMetadataRenderer
+                  prompt={prompt}
+                  schema={promptMetadataSchema}
                 />
               </Tabs.Panel>
-            )}
-          </Tabs>{" "}
-        </Container>
+
+              {checkParametersSupported(prompt) && (
+                <Tabs.Panel value="parameters">
+                  <ParametersRenderer
+                    initialValue={getPromptParameters(prompt)}
+                    onUpdateParameters={onUpdateParameters}
+                  />
+                </Tabs.Panel>
+              )}
+            </Tabs>
+          </Container>
+          <RunPromptButton prompt={prompt} size="full" />
+        </>
       ) : (
-        <ActionIcon size="sm" onClick={() => setIsExpanded(true)}>
-          <IconClearAll />
-        </ActionIcon>
+        <Flex direction="column" justify="space-between" h="100%">
+          <Flex direction="row" justify="center" mt="0.5em">
+            <ActionIcon size="sm" onClick={() => setIsExpanded(true)}>
+              <IconClearAll />
```

Comment:
Sorry noob question, which button is this from the UI? It's not clear to me right away?

Reply:
<img width="55" alt="Screenshot 2023-12-27 at 11 59 15 AM" src="https://github.com/lastmile-ai/aiconfig/assets/5060851/81f3da22-d9c5-42a4-bc3d-9288d3ddbbdb">

It's the stacked lines to open/close the drawer